src/components/marketplace/views.tsx

```diff
@@ -0,0 +1,456 @@
+  readonly encrypted?: boolean;
+  readonly highlyAvailable?: boolean;
+  readonly highIOPS?: boolean;
+  readonly memory?: {
+    readonly amount: number;
+    readonly unit: string;
+  };
+  readonly storage?: {
+    readonly amount: number;
+    readonly unit: string;
+  };
+}
+
+interface IMarketplaceItemPageProperties {
+  readonly linkTo: RouteLinker;
+  readonly service: IV3Service<IPaaSServiceMetadata>;
+  readonly plans: ReadonlyArray<IV3ServicePlan<IPaaSServicePlanMetadata>>;
+  readonly version?: string;
+  readonly versions: ReadonlyArray<string>;
+}
+
+interface IMarketplacePageProperties {
+  readonly linkTo: RouteLinker;
+  readonly services: ReadonlyArray<IV3Service<IPaaSServiceMetadata>>;
+}
+
+interface IServiceDetails {
+  readonly [serviceLabel: string]: {
+    readonly description?: string;
+    readonly image: string;
+    readonly imageTitle: string;
+    readonly usecase?: ReadonlyArray<string>;
+  };
+}
+
+interface ILogoProperties {
+  readonly image: string;
+  readonly imageTitle: string;
+  readonly label?: string;
+}
+
+interface IPlanTabProperties {
+  readonly linkTo: RouteLinker;
+  readonly plans: ReadonlyArray<IV3ServicePlan<IPaaSServicePlanMetadata>>;
+  readonly serviceGUID: string;
+  readonly version?: string;
+  readonly versions: ReadonlyArray<string>;
+}
+
+interface ITabProperties {
+  readonly active: boolean;
+  readonly href: string;
+  readonly children: string;
+}
+
+interface ITableRowProperties {
+  readonly canBeEncrypted: boolean;
+  readonly canBeHighIOPS: boolean;
+  readonly canBeHighlyAvailable: boolean;
+  readonly data: ICustomMetadata;
+  readonly costs?: ReadonlyArray<{
+    readonly amount: {
+      readonly usd: number;
+    };
+    readonly unit: string;
+  }>;
+  readonly limitsConcurrentConnections: boolean;
+  readonly limitsMemory: boolean;
+  readonly limitsStorage: boolean;
+  readonly name: string;
+  readonly providesBackups: boolean;
+}
+
+const serviceDetails: IServiceDetails = {
+  'cdn-route': {
+    image: cdnLogo,
+    imageTitle: 'Amazon Web Services - CloudFront - Official Logo',
+  },
+  'postgres': {
+    image: postgresLogo,
+    imageTitle: 'PostgreSQL - Official Logo',
+  },
+  'mysql': {
+    image: mysqlLogo,
+    imageTitle: 'MySQL - Official Logo',
+  },
+  'aws-s3-bucket': {
+    image: s3Logo,
+    imageTitle: 'Amazon Web Services - S3 Bucket - Official Logo',
+  },
+  'redis': {
+    image: redisLogo,
+    imageTitle: 'Redis - Official Logo',
+  },
+  'elasticsearch': {
+    image: elasticsearchLogo,
+    imageTitle: 'ElasticSearch - Official Logo',
+  },
+  'influxdb': {
+    image: influxdbLogo,
+    imageTitle: 'InfluxDB - Official Logo',
+  },
+};
+
+export function Tab(props: ITabProperties): ReactElement {
+  const classess = ['govuk-tabs__list-item'];
+  if (props.active) {
+    classess.push('govuk-tabs__list-item--selected');
+  }
+
+  return (
+    <li className={classess.join(' ')}>
+      <a href={props.href} className="govuk-link">
```

Comment:
can we use the corrected class from this fix? https://github.com/alphagov/paas-admin/pull/692